tangelo/linq/gate.py

```diff
@@ -125,7 +125,10 @@ def __eq__(self, other):
 
         ds, do = self.__dict__, other.__dict__
 
-        if any(ds[k] != do[k] for k in ds if k != "parameter"):
+        # Check CNOT CX Separately
```

Comment:
Can you be a bit more descriptive with that comment ?
What this code means to me is "Consider CX and CNOT gates to be the same"